tests/callbacks/test_early_stopping.py

```diff
@@ -294,3 +294,25 @@ def validation_epoch_end(self, outputs):
         (trainer.global_step, max(min_steps, by_early_stopping, by_min_epochs), step_freeze, min_steps, min_epochs)
 
     _logger.disabled = False
+
+
+def test_early_stopping_mode_options():
+    with pytest.raises(MisconfigurationException, match="unknown_option"):
+        EarlyStopping(mode="unknown_option")
+
+    with pytest.warns(DeprecationWarning, match="'auto' is deprecated"):
+        EarlyStopping()
+
+    with pytest.warns(DeprecationWarning, match="'auto' is deprecated"):
+        EarlyStopping(mode="auto")
+
+
+def test_model_checkpoint_mode_options():
+    with pytest.raises(MisconfigurationException, match="unknown_option"):
+        ModelCheckpoint(mode="unknown_option")
+
+    with pytest.warns(DeprecationWarning, match="'auto' is deprecated"):
+        ModelCheckpoint()
+
+    with pytest.warns(DeprecationWarning, match="'auto' is deprecated"):
+        ModelCheckpoint(mode="auto")
```

Comment:
added in
https://github.com/PyTorchLightning/pytorch-lightning/blob/410d67fbe866ee20069e88db1be729d27ae0af48/tests/deprecated_api/test_remove_1-3.py#L37-L38